src/components/ChatLog.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+
+const ChatLog = ( props ) => {
+
+    const [entries, setEntries] = useState(props.entries);
+
+    const setLiked = (id, value) => {
+        let numLikes = false;
+        entries.forEach(element => {
+            if(element.id === id){
+            element.liked = value;
```

Comment:
This is what caused your tests to fail. You're actually modifying state when you set `entry.liked`!